pmlc/runtime/executable.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Intel Corporation
```

Comment:
This doesn't seem to be runtime-specific. I think certain runtimes will have certain Executable implementations, but the interface should remain on the compiler.